CMakeLists.txt

```diff
@@ -125,7 +125,8 @@ endif()
 ############################################################
 # OPTION - MIOpen Backend
 # - OpenCL
-# - HIP
+# - HI
+message("HAS_HIP == " ${HAS_HIP})
```

Comment:
Maybe this change is not necessary? Or use VERBOSE or DEBUG message mode.

Reply:
accidentally touched CMakeLists.txt

Reply:
@kikimych Please do not forget to revert ;)